index.js

```diff
@@ -1219,9 +1223,9 @@
 
       $pairs.push ([x, y]);
       try {
-        return Setoid.test (x) &&
-               Setoid.test (y) &&
```

Comment:
This was actually a low-hanging performance improvement that I think we missed. There's no need to Setoid.test the second value, because it will never be asked to provide a `fantasy-land/equals` method, and we already predetermined that the two values are of the same type.

Reply:
![latest-screenshot](https://user-images.githubusercontent.com/1217745/115955675-4b790880-a4f8-11eb-81f5-8ad6f02c5e3b.png)

Reply:
That's compared to `master`.